src/content/libraries/async/async-await.md

```diff
@@ -1076,6 +1071,19 @@ bool _logoutSucceeds = false;
 
 </details>
 
+## Which lints work for futures?
+
+In real-world scenarios, enable the following two lints to help with
+futures.
```

Comment:
"In real-world scenarios" doesn't add any meaning here. Instead I'd introduce the topic as:

```suggestion
To catch common mistakes that arise while working with async and futures,
[enable](/tools/analysis#individual-rules) the following lints:
```